src/Microsoft.VisualStudio.VsInteractiveWindow/VsInteractiveWindowCreationParameters.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using Microsoft.VisualStudio.OLE.Interop;
+using Microsoft.VisualStudio.Shell.Interop;
+
+namespace Microsoft.VisualStudio.InteractiveWindow.Shell 
+{
+    /// <summary>
+    /// Provides creation parameters to the interactive window factory.
+    /// </summary>
+    public class VsInteractiveWindowCreationParameters 
```

Comment:
I started with struct, but then looked at other .NET like ProcessStartupInfo and similar and they appear to be classes